awx/main/models/inventory.py

```diff
@@ -58,6 +58,16 @@
 logger = logging.getLogger('awx.main.models.inventory')
 
 
+class InventoryConstructedInventoryMembership(models.Model):
+    constructed_inventory = models.ForeignKey('Inventory', on_delete=models.CASCADE, related_name='constructed_inventories')
```

Comment:
I think it might not be a good idea to put a `related_name` on the field on a through model. Think about `inventory.constructed_inventories.all()`, you would expect this to be a list of `Inventory` objects, but if the related_name is defined here, that would be a list of `InventoryConstructedInventoryMembership`, which is :exploding_head: